craft_parts/sequencer.py

```diff
@@ -190,30 +193,26 @@ def _run_step(
             state = states.PullState(
                 part_properties=part_properties,
                 project_options=self._project_info.project_options,
-                assets={},  # TODO: obtain pull assets
             )
 
         elif step == Step.BUILD:
             state = states.BuildState(
                 part_properties=part_properties,
                 project_options=self._project_info.project_options,
-                assets={},  # TODO: obtain build assets
+                overlay_hash=self._layer_state.get_overlay_hash().hex(),
```

Comment:
What happened with `assets`?

Reply:
This was originally based on Snapcraft states and craft-parts doesn't use them for lifecycle planning. The need for asset listing will be re-evaluated during integration with next-gen snapcraft, when we can reinstate it or drop it if not needed